mobile/lib/modules/sharing/ui/selection_thumbnail_image.dart

```diff
@@ -131,27 +131,26 @@ class SelectionThumbnailImage extends HookConsumerWidget {
               child: _buildSelectionIcon(asset),
             ),
           ),
-          asset.type == 'IMAGE'
-              ? Container()
-              : Positioned(
-                  bottom: 5,
-                  right: 5,
-                  child: Row(
-                    children: [
-                      Text(
-                        asset.duration.toString().substring(0, 7),
-                        style: const TextStyle(
-                          color: Colors.white,
-                          fontSize: 10,
-                        ),
-                      ),
-                      const Icon(
-                        Icons.play_circle_outline_rounded,
-                        color: Colors.white,
-                      ),
-                    ],
+          if (asset.type != 'IMAGE')

```

Comment:
I've been seeing you are using this a lot and I have to say, it really does reduce the amount of empty widget build :)